src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -310,6 +321,15 @@ internal IEnumerable<V> Search(string query, int minResultsForTolerantSearch = 0
             query = query.ToLower();
 
             var subPatterns = SplitOnWhiteSpace(query);
+
```

Comment:
Just as a question; should this code be wrapped in #if DEBUG statements? Probably not necessary, but I am doing the same type of code and wanted to get the group thought about if it is needed/wanted or not.

Reply:
I think because this code is already in the debug only menu it's fine not to use a compiler flag. On the other hand, for performance impacting code - like `Console.WriteLine` I would use a debug flag:
https://stackoverflow.com/questions/18464833/console-writeline-effect-on-performance